formats/ome-tiff/specification.txt

```diff
@@ -1,26 +1,27 @@
 OME-TIFF specification
 ======================
 
-
 The following provides technical details on the OME-TIFF
 format. It assumes familiarity with both the
 `TIFF <http://partners.adobe.com/public/developer/tiff/index.html>`_ and
 :schema_plone:`OME-XML <>` formats, although there is some review of both.
 
-Storing OME-XML within a TIFF
------------------------------
+An OME-TIFF dataset consists of;
+
+- one or more files in standard TIFF format with the file extension
+  ``.ome.tif`` or ``.ome.tiff`` or `BigTIFF format <http://bigtiff.org/>`_
+  with the file extension ``.ome.tf2``, ``.ome.tf8`` or ``.ome.btf``
```

Comment:
Note that 'ome.tif' and 'ome.tiff' are also valid when using big tiff; we might want to clarify that here.